### ISSUE TITLE
FL0034: False positive with C# Action<XXX>

### ISSUE THREAD

**Issue by Evangelink** [fsprojects/FSharpLint#481]
### Description

Call a C# method expecting a parameter of type `Action<XXX>` with a fsharp fun and you will get the false-positive.

### Repro steps

For example, when using `Python.Included`, do:

```fsharp
Installer.add_LogMessage (fun x -> Log.Debug x) // FL0034
//Installer.add_LogMessage (Log.Debug) // Code is invalid
```

### Expected behavior

No issue reported as call-site doesn't allow such code.

### Actual behavior

Warning raised.

### Known workarounds

Not aware of any workaround.

### Related information

* Operating system: Win10
* Branch: VS extension
* .NET Runtime: 4.8
